local-certs/server.nk

```diff
@@ -0,0 +1 @@
+SUAIUWTN33PU3ASNU2YT5MRQTITBQXKO5JDBL2K4IJ4PN2SPZBOO2AAYSQ
```

Comment:
In customer deployments, this .nk file will hold the customer's NKey private key from their own key pair and will be access controlled via the file system.

Reply:
This will be the interesting piece we will need to implement. We will need to store the private keys as a vault file or similar on the FS.